x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.tsx

```diff
@@ -210,6 +210,9 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
     mlSuppressionFields,
   } = useMLRuleConfig({ machineLearningJobId });
 
+  const isMlSuppressionIncomplete =
+    isMlRule(ruleType) && machineLearningJobId?.length && !allJobsStarted;
```

Comment:
returning number in boolean variable can lead to rendering `0`

<img width="1218" alt="Screenshot 2024-07-25 at 11 29 20" src="https://github.com/user-attachments/assets/94566f39-6f4c-49ba-bef9-2718aed8f005">

Reply:
Well, that's embarrassing 😅 . Addressed in [db1d4b0](https://github.com/elastic/kibana/pull/189003/commits/db1d4b02cc8ab0b5ff567bcaa3e9de61e1124463)